cmd/payload-testing-ui/server.go

```diff
@@ -126,7 +128,18 @@ type server struct {
 }
 
 func newServer(client ctrlruntimeclient.Client, ctx context.Context, namespace string) (server, error) {
-	runsListTemplate, err := template.New("runsListTemplate").Parse(runsListTemplate)
+	runsListTemplate, err := template.New("runsListTemplate").Funcs(template.FuncMap{
+		"runLink": func(obj *metav1.ObjectMeta, pr *prpqv1.PullRequest) template.HTML {
+			objName := obj.GetName()
+			objNS := obj.GetNamespace()
```

Comment:
```suggestion
			objName := obj.Name
			objNS := obj.Namespace
```

similarly in other places.